add-slug-to-blog-posts.js

```diff
@@ -0,0 +1,21 @@
+// Adds `slug: '{filename-without-extension}'` to all blog posts in /blog
+
+import globby from 'globby'
+import { basename } from 'path'
+import { readFile, writeFile } from 'node:fs/promises'
+
+const paths = await globby('./blog/*.md')
+
+for (const path in paths) {
+  const file = await readFile(paths[path], { encoding: 'utf-8' })
+  
+  let slug = basename(paths[path], '.md')
+    .replace(/-/g, '/')
+  console.log(slug)
+
+  const fileWithSlug = file.replace(/\n---/, `\nslug: "${slug}"\n---`)
+  console.log(fileWithSlug)
+  await writeFile(paths[path], fileWithSlug, { encoding: 'utf-8' })
+}
+
+// console.log(paths)
```

Comment:
```suggestion
```

Reply:
It's a one-off script and for me personally it's easier to uncomment console.log to debug something, since it hints at how i did it when I wrote the script. Confusing explanation, but I tried!